tests/features/tmgmt/tmgmt_dgt_connector_cart.feature

```diff
@@ -151,10 +151,11 @@ Feature: TMGMT Poetry Cart features
     And I wait for AJAX to finish
     And I fill in "Date" with a relative date of "+20" days
     And I press "Submit to translator"
+    And I wait 30 seconds
     Then I should see the message "Job has been successfully sent for translation."
     And I should see text matching "Vocab \(taxonomy\:vocabulary\:\d\) and 1 more"
 
-  @javascript
+  @javascript @maximizedwindow
```

Comment:
I think we should try to avoid this tag like the plague.

We found a pretty nice workaround to get around this here: https://github.com/ec-europa/platform-dev/pull/2481/files#diff-d598cdde109611da2beae3a6b49783bfR438

Reply:
cc @Fefaine